packages/react-router-dom/package.json

```diff
@@ -23,12 +23,14 @@
     "index.js",
     "matchPath.js",
     "withRouter.js",
-    "README.md",
```

Comment:
Reverted. I figured it was safe to remove since there are other automatically-included files that aren't explicitly listed.